docs/index.md

```diff
@@ -5,12 +5,14 @@
    :caption: User Guides
    :maxdepth: 1
 
-   userguides/quickstart
+   userguides/config
+   userguides/console
    userguides/projects
    userguides/plugins
-   userguides/config
+   userguides/quickstart
```

Comment:
Why is quickstart so far down?

p.s. the ordering here matters in terms of what you see first

Reply:
Reverted the changes to main branch and added testing to the bottom